translation-config.yml

```diff
@@ -1,6 +1,46 @@
+locales:
+  default: en_US
+  supported:
+    - ar
+    - de
+    - en_GB
+    - es
+    - fr_CA
+    - fr_FR
+    - it
+    - ja
+    - nl
+    - pl
+    - pt
+    - ru
+    - sv
+    - zh_CN
+    - zh_HK
+
 bundles:
   - path: .
     extension: properties
     default_locale: en_US
   - path: src/main/resources
     extension: json
+
+snapshot:
+  copy:
+    - en_GB
+
+io:
+  out:
+    generator: translations.extensions.io_xls.XlsExporter
+    name: translations
+    mapping:
+      fr_CA: fr
+      fr_FR: fr
+      en_US: ~
+      en_GB: ~
+  in:
+    importer: translations.extensions.io_xls.XlsImporter
```

Comment:
Instead of having to directly list this out can we make it a bit more easy to consume by just specifying the importer's name? Something like `importer: XlsImporter` would be much more readable and usable and we can do the parsing of what this means in the script itself

Reply:
I thought about it, but though of cases where you would implement your own importer/exporter, and name clashes, thus the reason for using FQN. Now, as an alternative, it can support both; that way you can configure it using just the name, and if there is a naming clash, then you would have to provide the FQN.